### ISSUE TITLE
Some of my Spotify Playlists are missing on Sesame

### ISSUE THREAD

**Issue by mrduud** [BranchMetrics/sesame_issues#332]
It used to be that ALL my Spotify playlists appeared on Sesame (S10e phone) but now any new playlists don't appear on Sesame. I am pressing "Settings > Refresh all shortcut data".

I uninstalled and reinstalled Sesame, and it now seems to be working properly on my brand new S23 phone. I tried the same on my S10e phone and yet the same problem persists.

When re-instaling Sesame:

- S23: the app started brand new and I had to set it up again from scratch

- S10e: the app opened as if it was already set up. I even tried clearing the cache and storage before I uninstalled it and the same thing happened.

When I go into shortcuts, I note that on my S23 it says I have 345 shortcuts under Spotify, and in my s10e it says I have 285. I want to use my S10e as my music phone so this is really important to me a I do a lot of DJ work from my phone.

TLDR: How can I get Sesame to view ALL my Spotify playlists on my s10e?

**Comment by mrduud** [comment]
I resolved it by resetting my S10e phone to factory settings.

**Comment by mrduud** [comment]
Further update: The problem came back soon on my S10e. So I can only assume that Sesame is not supported on the S10e amymore , which means I can't use it as my music phone. Basically I can only make homscreen buttons from existing playlists. It won't recognise any new playlists that I make.
